src/Core/Content/Feeds/Product/template/body.xml.twig

```diff
@@ -58,5 +58,5 @@
 
     {% set gender = product.properties.elements|filter(property => property.group.name == 'Gender')|first %}
     <gender>{% if gender %}{{ gender.name }}{% endif %}</gender>
-    <instock>{% if product.availableStock > 0 %}true{% else %}false{% endif %}</instock>
+    <instock>{% if product.stock > 0 %}true{% else %}false{% endif %}</instock>
```

Comment:
@wexoigi it should be availableStock since that represents the stock minus purchases by customers, meaning the actual amount of products that are available